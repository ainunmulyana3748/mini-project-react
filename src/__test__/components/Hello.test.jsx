import { render } from "@testing-library/react";
import Hello from "../../components/Hello";

test("snapshot test", () => {
  const { container } = render(<Hello name="Dian" />);
  expect(container).toMatchSnapshot();
});
