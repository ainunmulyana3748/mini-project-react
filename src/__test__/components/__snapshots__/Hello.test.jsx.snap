// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<div>
  <h1>
    Hello, 
    Dian
    !
  </h1>
</div>
`;
